src/python/benchUtil.py

```diff
@@ -1326,10 +1327,27 @@ def simpleReport(self, baseLogFiles, cmpLogFiles, jira=False, html=False, baseDe
       qpsCmp = avgQPSCmp
 
       # print '%s: %s' % (desc, abs(qpsBase-qpsCmp) / ((maxQPSBase-minQPSBase)+(maxQPSCmp-minQPSCmp)))
-      # TODO: need a real significance test here
       if qpsStdDevBase != 0 or qpsStdDevCmp != 0:
-        significant = (abs(qpsBase-qpsCmp) / (2*qpsStdDevBase+2*qpsStdDevCmp)) > 0.30
+        assert len(baseQPS) == len(cmpQPS)
+
+        # Student's T-Test is often used for distributions with the same underlying variance and number of samples, but
+        # with large number of samples, Student's T distribution approaches a normal distribution
+
+        # the "combined" std.dev. is the square root the geometric mean of the two variances
+        qpsStdDev = math.sqrt(qpsStdDevBase * qpsStdDevBase + qpsStdDevCmp * qpsStdDevCmp)
```

Comment:
Hmm the math does not need to see how many samples of each A/B experiment we drew?

Reply:
It does ... see below where we divide by len(baseQPS)